mapdata/WarcraftLegacies/AbilityData/Skin/1245271123.json

```diff
@@ -0,0 +1,119 @@
+{
+  "OldId": 1634693697,
+  "NewId": 1245271123,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 2020631137,
+      "Value": 3
+    },
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(tess)"
+    },
+    {
+      "Id": 1835101793,
+      "Type": 3,
+      "Value": "Vanish"
+    },
+    {
+      "Id": 1918988897,
+      "Type": 3,
+      "Value": "ReplaceableTextures/CommandButtons/BTNAbility_Vanish.blp"
+    },
+    {
+      "Id": 2020635233,
+      "Value": 3
+    },
+    {
+      "Id": 1802007137,
+      "Type": 3,
+      "Value": "R"
+    },
+    {
+      "Id": 1952805473,
+      "Type": 3,
+      "Value": "Learn Vanish- [|cffffcc00Level %d|r]"
+    },
+    {
+      "Id": 1953854049,
+      "Type": 3,
+      "Value": "The hero vanishes, becoming invisible and gains 20 bonus damage. The invisibility will not dissipate even if the hero attacks or casts spells.|n|n|cffffcc00Level 1|r - 10 seconds duration|n|cffffcc00Level 2|r - 15 seconds duration|n|cffffcc00Level 3|r - 20 seconds duration"
+    },
+    {
+      "Level": 1,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Vanish - [|cffffcc00Level 1|r]"
+    },
+    {
+      "Level": 2,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Vanish - [|cffffcc00Level 2|r]"
+    },
+    {
+      "Level": 3,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Vanish - [|cffffcc00Level 3|r]"
+    },
+    {
+      "Level": 4,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Time\u0027s Shield - [|cffffcc00Level 4|r]"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "The hero vanishes, becoming invisible and gaining 20 damage for 10 seconds. The invisibility will not dissipate even if the hero attacks or casts spells."
```

Comment:
Generate all the numbers here from object data. Let me know if you need help with learning how to do that.